feat(ins): add information for eol api (#4206)
scaleway/docs-content: compute/instances/how-to/migrate-volumes-snapshots-to-sbs.mdx

  paragraph: This guide provides step-by-step instructions to help you migrate your existing Block Storage volumes and snapshots to Scaleway's new Block Storage management system.
tags: instance snapshot volume b_ssd sbs
dates:
  validation: 2025-01-14
  posted: 2024-08-07
categories:
  - compute
---

To enhance performance and reliability, Scaleway is transitioning the management of Block Storage volumes and snapshots from Compute to Storage.

To facilitate the transition to Scaleway's new Block Storage management, two main approaches for migrating your volumes exist, each with its unique benefits and drawbacks:

- [Using the Instance API/CLI migration endpoint (Plan and Apply)](#migrating-using-the-migration-endpoint-of-the-instances-api-cli):
   - **Advantage:** Migration can occur without service interruption, as the volume remains attached to the running Instance and fully accessible.
   - **Drawback:** Only the volume's representation is migrated to the SBS API; the data itself doesn't move. This means the migrated volume will not benefit from low latency or higher IOPS, as it retains the original `b_ssd` characteristics.

> - [Using the snapshot export/import features](#migrating-using-the-snapshot-exportimport-features):

Reviewer: Shouldn't there be a slash in the anchor, as the title (## Migrating using the snapshot import/export features) does have it?

Author: Fixed anchor, no slash required after testing

   - **Advantage:** The migrated volume will fully support low latency, and it is possible to modify the IOPS up to 15k. All existing data of the old volume will be copied to the new Block Storage Low Latency volume.

>    - **Drawback:** A snapshot must be manually created, exported, and then imported and attached to the Instance which may require downtime.

Reviewer 2: maybe mention after the snapshot is imported a volume should be created from this snapshot (it's a volume that is attached to an instance not a snapshot)

>    - **Drawback:** A snapshot must be manually created, exported, and imported. After importing, the snapshot must be converted into a volume, which then needs to be attached to the instance. This process may require downtime.

Author: ```suggestion
   - **Drawback:** A snapshot must be manually created, exported, and imported. After importing, the snapshot must be converted into a volume, which must then be attached to the Instance. This process may require downtime.
```

<Macro id="block-storage-no-migration-for-k8s" />

<Macro id="requirements" />

- A Scaleway account logged into the [console](https://console.scaleway.com)
- An [Instance](/compute/instances/how-to/create-an-instance/) using [Block Storage volumes](/storage/block/how-to/create-a-volume/)

<Message type="tip">
  - The easiest way to migrate your Block Storage volumes and snapshots is by using the [Scaleway Command Line Interface (CLI)](/developer-tools/scaleway-cli/quickstart/) tool.
  - If you encounter an error about quota limitations during the migration process, [contact our support team](https://console.scaleway.com/support/tickets/create) for assistance.
</Message>

<Message type="important">
  To ensure continued access to managing your volumes, you must convert your **unified volumes and snapshots** into either **local** or **Block volumes** before the **end-of-life date of June 2nd, 2025**. After this date, unified volumes and snapshots will no longer be supported by any API.

  #### Key migration deadlines:
  - April 30th, 2025:
    - Creation of new unified volumes via the Instance API will be disabled.

>     - Unified volumes will no longer be supported by the Block Storage API.

Reviewer 3: ```suggestion
    - Unified volumes will no longer be supported by the Instance API.
```

  </TabsTab>
</Tabs>

## Migrating using the snapshot import/export features

To convert your unified volumes and snapshots you can use the snapshot export/import feature as an alternative solution to transition a Block Storage legacy volume into a Block Storage Low Latency volume.

    <Message type="important">
      The process below requires manual intervention and **does not preserve the original volume’s ID**. While this approach allows you to transition to a Block Storage Low Latency volume, it is only a workaround.
    </Message>

   Steps:

>    1. [Export a snapshot of your unified volume](https://www.scaleway.com/en/docs/compute/instances/api-cli/snapshot-import-export-feature/#exporting-snapshots).
>    2. [Import the snapshot into a new local or Block volume](https://www.scaleway.com/en/docs/compute/instances/api-cli/snapshot-import-export-feature/#importing-snapshots).

Reviewer: ```suggestion
   1. [Export a snapshot of your unified volume](/compute/instances/api-cli/snapshot-import-export-feature/#exporting-snapshots).
   2. [Import the snapshot into a new local or Block volume](/compute/instances/api-cli/snapshot-import-export-feature/#importing-snapshots).
```
